applications/wg-easy/container/Containerfile

```diff
@@ -0,0 +1,68 @@
+# Base image for all shared Dockerfiles for taskfiles
+# Use this image as base image for app specific docker files
+FROM --platform=$BUILDPLATFORM ubuntu:24.04
+
+ARG TARGETOS
+ARG TARGETARCH
+
+WORKDIR /tools
+
+# Set environment variables
+ENV DEBIAN_FRONTEND=noninteractive \
+    HOME=/home/devuser \
+    SHELL=/bin/bash
+
+# Install CLI tools
+RUN apt-get update && apt-get install -y \
+    curl \
+    jq \
+    yq \
+    gnupg \
+    sudo \
+
+    # Install Helm
+    && curl https://raw.githubusercontent.com/helm/helm/main/scripts/get-helm-3 | bash \
```

Comment:
Why is this all one big RUN line?
Doesn't this mean that you're forcing the cache to be invalidated if _any_ of these things change rather than only updating the one change? Is there a benefit to it b/c I only see a downside of breaking normal caching behavior.

Reply:
The image size I believe is driven exclusively off the fact that you include `&& rm -rf /var/lib/apt/lists/*` in the same RUN command as the `apt install` and that I'm aware of and fine with.

What I don't think you need to be doing is mixing apt and curl installs. Using all of the apt installs in a single RUN and ending it with the cleanup is good practice. Grouping independent `curl` commands inside of an apt RUN command is what I would break up here.

Reply:
I've split the RUN directives to separate debian package installs and downloading tools